dbt_subprojects/tokens/models/prices/prices_trusted_tokens.sql

```diff
@@ -61,6 +61,7 @@ WITH trusted_tokens AS (
                 , ('ethereum', 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599)
                 , ('ethereum', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e)
                 , ('fantom', 0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83)
+                , ('flare', 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d)
```

Comment:
are we waiting for erc20 transfer table to exist to find top tokens? or finding some via flare explorer?

Reply:
I have updated the known tokens, and this may still be an open question.
We may need to choose a few to fill in this table.

Reply:
lets revisit when we add DEXs (if applicable)
we should do in a separate PR first, so we can get live in prod prior to building downstream spells that use it